hedera-mirror-rest/transactions.js

```diff
@@ -178,6 +181,24 @@ const createNftTransferList = (nftTransferList) => {
   });
 };
 
+/**
+ * Creates a custom fee limit transfer list from an aggregated array of JSON objects in the query result
+ *
+ * @param {Object[]} maxCustomFeesList - The list of max custom fees
+ * @return {CustomFeeLimitViewModel[]} An array of custom fee limit view models
+ */
+const createMaxCustomFeesTransferList = (maxCustomFeesList) => {
+  if (!maxCustomFeesList || maxCustomFeesList.length === 0) {
+    return [];
+  }
+
+  return maxCustomFeesList.map((fee) => {
+    const feeBuffers = Array.isArray(fee) ? fee.map((f) => Buffer.from(f)) : [Buffer.from(fee)];
+    const customFeeLimit = new CustomFeeLimit(feeBuffers);
+    return new CustomFeeLimitViewModel([customFeeLimit]);
+  });
```

Comment:
`maxCustomFeesList` is either null or an array of js `Buffer`s. Model `CustomFeeLimit` (I should should rename it to CustomFeeLimits` should accept the array of `Buffer`s as input, then view model `CustomFeeLimitsViewModel` should convert it to the view model. Assume `CustomFeesLimit` model handles both null and empty array, this can be simplified to just

```suggestion
  return new CustomFeeLimitsViewModel(new CustomFeeLimits(maxCustomFeesList));
```